samples/Samples.Wcf/Program.cs

```diff
@@ -10,6 +10,11 @@ public static class Program
         private const string WcfPort = "8585";
         private const string WcfNamespace = "WcfSample";
 
+#if NETFRAMEWORK
```

Comment:
Can we remove this change for now since we're not touching the Wcf integration in this PR?

Reply:
sure.